person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @age >= 18 || parent_permission
+  end
+
+  private :is_of_age?
```

Comment:
- [ ] the method name is `of_age?` to check the age, Here you have set the method `is_of_age?` private. Please make sure to correct the method name here.